packages/lumx-react/src/components/badge/Badge.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { mdiHeart } from '@lumx/icons';
+import { AspectRatio, Icon, Size, Thumbnail, ThumbnailVariant } from '@lumx/react';
+import { decorators } from '@lumx/react/story-block';
+import { text } from '@storybook/addon-knobs';
+import React from 'react';
+import { ColorPalette } from '..';
+import { Badge } from './Badge';
+
+export default { title: 'Badge', decorators };
+
+export const badgeValue = () => (
+    <Badge color={ColorPalette.blue}>
```

Comment:
It can be fun to change the color directly from storybook with the "select" knob